app/tests/spec/views/sign_up.js

```diff
@@ -624,6 +626,30 @@ function (chai, _, $, moment, sinon, p, View, Session, AuthErrors, Metrics,
           });
       });
 
+      it('passes the customize sync option to the fxa-client', function () {
+        relier.set('service', 'sync');
+        relier.set('customizeSync', true);
+
+        sinon.stub(view.fxaClient, 'signUp', function () {
+          return p({
+            verified: true
+          });
+        });
+
+        sinon.stub(broker, 'afterSignIn', function () {
+          return p();
+        });
+
+        return view.render()
+          .then(function () {
+            fillOutSignUp(email, 'password', { year: CURRENT_YEAR - 14, context: view });
+            return view.submit();
+          })
+          .then(function () {
+            assert.isTrue(view.fxaClient.signUp.calledWith(email, 'password', relier,
```

Comment:
I added a check for `afterSignIn` here: https://github.com/mozilla/fxa-content-server/pull/2021/files#diff-a751175eb826ca71a4a1fbb4b30f3364R487